analyticsdataserver/settings/local.py

```diff
@@ -66,6 +74,9 @@
 
 ANALYTICS_DATABASE = 'analytics'
 ENTERPRISE_REPORTING_DB_ALIAS = 'analytics'
+# Requests to the /v1/ api will use this database.
+# To test a second database under the /v1/ routes change this to 'analytics_v1'
+ANALYTICS_DATABASE_V1 = 'analytics'
```

Comment:
Before we merge, shall we just update this to `analytics_v` so it does not give the false impression we already have the new DB populated?